tests/cloner_test.go

```diff
@@ -299,8 +303,10 @@ var _ = Describe("Validate creating multiple clones of same source Data Volume",
 		dvs := []*cdiv1.DataVolume{targetDv1, targetDv2, targetDv3}
 
 		for _, dv := range dvs {
-			_, err = utils.CreateDataVolumeFromDefinition(f.CdiClient, f.Namespace.Name, dv)
+			dataVolume, err := utils.CreateDataVolumeFromDefinition(f.CdiClient, f.Namespace.Name, dv)
 			Expect(err).ToNot(HaveOccurred())
+			f.ForceBindPvcIfDvIsWaitForFirstConsumer(dataVolume)
```

Comment:
Looks like this assumes that DV name = PVC name? If so can you add a comment that this needs to get fixed when we merge the PR that makes this assumption invalid? This happens in numerous places, please put a comment on all of them so we can easily identify them.

Reply:
very good catch.

fortunately this is enclosed inside ForceBindPvcIfDvIsWaitForFirstConsumer so only has to be fixed once, I have added comment inside the function

the other function -  ForceBindIfWaitForFirstConsumer just uses pvc, so the code to get pvc was already in the test.